pandas/core/series.py

```diff
@@ -5993,6 +5993,58 @@ def mask(
 
     index = properties.AxisProperty(
         axis=0, doc="The index (axis labels) of the Series."
+            """
+            
+            A one-dimensional ndarray with hashable type axis labels. 
+
+            Pandas Series.index attribute is used to get or set the index labels 
+            of the given Series object.The object supports both 
+            integer- and label-based indexing and provides a host of methods for 
+            performing operations involving the index. 
+            The labels need not be unique but must be of the hashable type.
+            
+            Returns
+            -------
+            index
+                index of the series
+
+            Examples
+            --------
+            **Use the Series.index attribute to set the index label for 
+            the given Series object.** 
+
+            >>> s = pd.Series(['Kolkata', 'Chicago', 'Toronto', 'Lisbon'])
```

Comment:
This example is not very realistic and confusing with the `City 1`, `City 2` labels. Maybe something like using the city as index and then making the value a feature like population, average temperature... would make more sense.

Also, since here you're just showing the `.index` accessor, better create the index with `Series(..., index=...)` and simply show that `your_series.index` returns the index.